CI-CD/Jenkins/Jenkins_pipeline_docker_scan.groovy

```diff
@@ -45,6 +45,11 @@ pipeline {
             docker images | grep myImage | awk '{ print \0443 }'
             """
             ).trim()
+            WS_DOCKER_SCANIMAGES = true
```

Comment:
@tabac-mend it is still missing WS_DOCKER_INCLUDES=name-of-your-docker-image-here

Reply:
@tabac-mend Can you put some comments in the script that explain what it is doing.  It is finding the image id by the name.   This would be helpful for those that are unfamiliar with jenkins, docker, etc.